qml/pages/FirstPage.qml

```diff
@@ -41,67 +62,91 @@ Page {
             }
         }
 
-        SilicaListView {
-            id: listView
-            model: []
-            anchors.fill: parent
-            header: PageHeader {
-                title: qsTr("Current Broadcasts")
+        onModelChanged: rowCount = Math.ceil(model.length / columnCount)
```

Comment:
Should this also be `gridView.model` for clarity? Most likely works since the model is bound inside the event handler I think.

Reply:
I don't think it's necessary; we are inside onModelChanged, which is inside the gridView, too. I would leave this as-is.